### PR TITLE
[KIECLOUD-97] -  Propagate OpenShiftStartupStrategy related changes t…

### PR DESCRIPTION
**Pull request by spolti**
…o KIE APB, and Operator

unit tests for hook script

Signed-off-by: Filippe Spolti <fspolti@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[KIECLOUD-XYZ] Subject`, `[RHDM-XYZ] Subject` or `[RHPAM-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: jboss-kie-common/added/launch/jboss-kie-common.sh [hunk 1]
```diff
@@ -7,13 +7,13 @@ query_ocp_api() {
     local ocpApi="${1}"
     local resourcePath="${2}"
     # only execute the following lines if this container is running on OpenShift
-    if [ -e /var/run/secrets/kubernetes.io/serviceaccount/token ]; then
+    if [ -e /var/run/secrets/kubernetes.io/serviceaccount/token ] || [ "${K8S_ENV}" = true ] ; then
```
Comment (sutaakar) [inline jboss-kie-common/added/launch/jboss-kie-common.sh]: Is this change required considering that token is created in bats test (just curious, I am not familiar with functionality of bats tests).

Comment (spolti) [reply jboss-kie-common/added/launch/jboss-kie-common.sh]: You're right, I forgot to delete it, it was letf prior the previous tests I did.
